app/src/main/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -421,7 +426,7 @@ public void onClick(View v) {
      * Accept an incoming Call
      */
     private void answer() {
-        activeCallInvite.accept(this, callListener);
+        activeCallInvite.accept(this, new AcceptOptions.Builder().build(), callListener);
```

Comment:
Let's use the regular `accept(Context context, Call.Listener listener)` for this one. Since the builder doesn't provide any additional options.

Reply:
ok.